robinhood.spec.in

```diff
@@ -159,7 +157,7 @@ Annex tools for robinhood.
 %setup -q -n @PACKAGE@-%{version}
 
 %build
-./configure @ac_configure_args@ %{?configure_flags:configure_flags} \
```

Comment:
pb:
running  
`./configure --disable-lustre && make rpm`
builds robinhood with lustre support because it doesn't pass --disable-lustre to configure...